From 3c288da5eec81ee58b85927df18d9194ead8f5c2 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Tue, 12 Jun 2018 11:59:01 +0100
Subject: drivers/dri/i965: add missing #include

brw_bufmgr.h uses time_t without include time.h, so the build fails under musl.

Reviewed-by: Eric Engestrom <eric.engestrom@intel.com>

Downloaded from upstream commit
https://cgit.freedesktop.org/mesa/mesa/commit/src/mesa/drivers/dri/i965/brw_bufmgr.h?id=3c288da5eec81ee58b85927df18d9194ead8f5c2

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 src/mesa/drivers/dri/i965/brw_bufmgr.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/mesa/drivers/dri/i965/brw_bufmgr.h b/src/mesa/drivers/dri/i965/brw_bufmgr.h
index 8cdf944..32fc7a5 100644
--- a/src/mesa/drivers/dri/i965/brw_bufmgr.h
+++ b/src/mesa/drivers/dri/i965/brw_bufmgr.h
@@ -37,6 +37,8 @@
 #include <stdbool.h>
 #include <stdint.h>
 #include <stdio.h>
+#include <time.h>
+
 #include "util/u_atomic.h"
 #include "util/list.h"
 
-- 
cgit v1.1

